src/handlers/create-comment-embedding.ts

```diff
@@ -0,0 +1,14 @@
+import { CallbackResult } from "../proxy-callbacks";
+import { Context } from "../types";
+
+export async function createCommentEmbedding(context: Context<"issue_comment.created">): Promise<CallbackResult> {
+  const {
+    logger,
+    adapters: { supabase },
+  } = context;
+
+  const uploaded = await supabase.embeddings.createConversationEmbeddings(context.payload.comment.node_id, context.payload, "comment");
+  logger.ok(`Successfully created comment!`, { ...uploaded, embedding: "removed for brevity" });
+
+  return { status: 200, reason: "success" };
```

Comment:
What's the purpose of reason property 

200 is self explanatory

Reply:
I think going forward it would make more sense to use `status` and eventually `message` if we wanna convey data.

Reply:
We should copy other popular frameworks and use `statusCode`